sunpy/map/mapbase.py

```diff
@@ -742,18 +742,28 @@ def rsun_meters(self):
 
     @property
     def rsun_obs(self):
-        """Radius of the Sun."""
+        """
+        Angular radius of the Sun.
+
+        Notes
+        -----
+        This value is taken the ``'rsun_obs'``, ``'solar_r'``, or ``radius``
+        FITS keywords.
```

Comment:
Might as well expand this description to also note the fallback of using the observer coordinate.